_integration-schemas/darksky/foreign-keys.md

```diff
@@ -0,0 +1,21 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas-foreign-keys/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+tap-reference: "integration"
```

Comment:
I don't think this file is actually needed, since the integration only returns a single table. I'd say delete it.